app/templates/mock-login.hbs

```diff
@@ -0,0 +1,39 @@
+<div class="au-c-main-container" itemprop="mainContentOfPage">
+    <div class="au-c-main-container__content au-c-main-container__content--scroll">
+        <main id="content">
+            <div class="au-o-region-large">
+                <div class="au-o-layout">
+                    {{#if this.session.isAuthenticated}}
+                    Logged in
+                    <MockLogout />
+                    {{else}}
```

Comment:
This part isn't needed since a logged in user will be redirected to the index page already.